src/components/cylc/tree/TreeItem.vue

```diff
@@ -37,11 +37,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       <slot name="cyclepoint" v-if="node.type === 'cycle'">
         <div :class="getNodeDataClass()" @click="nodeClicked">
           <!-- NOTE: cycle point nodes don't have any data associated with them
-            at present so we must use the root family node for the task icon.
-            We don't use this for the v-cylc-object as that would set the node
-            type to family. -->
+            at present so we must use the root family node for the task icon. -->
           <Task
-            v-cylc-object="node"
+            v-cylc-object="node.familyTree[0]"
```

Comment:
This changes `<workflow>//<cycle>` to `<workflow>//<cycle>/root` for *all* mutations. I think this may break things.

It might be safer to handle this in the edit-runtime section of the code e.g:

```js
if (!tokens.task) {
  // this is a cycle point so we will operate on the "root" family
  tokens.duplicate({task: 'root'})
}
```